site/content/en/latest/user/http-timeouts.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "HTTP Timeouts"
+---
+
+The [HTTPRouteTimeouts][] resource allows users to configure request timeouts and response timeouts for an [HTTPRouteRule][]. This guide shows how to configure timeouts.
+
+The [HTTPRouteTimeouts][] supports two kinds of timeouts:
+- **request**: Request specifies the maximum duration for a gateway to respond to an HTTP request. 
+- **backendRequest**: BackendRequest specifies a timeout for an individual request from the gateway to a backend.
+
+__Note:__  The Request duration must be >= BackendRequest duration
+
+## Installation
+
+Follow the steps from the [Quickstart Guide](quickstart.md) to install Envoy Gateway and the example manifest.
+Before proceeding, you should be able to query the example backend using HTTP.
+
+## Verification
+
+Httpbin has the ability to delay responses; we use it as the backend to control response time.
```

Comment:
We recently updated the echo-server in use in EG to a version that supports delayed execution, e.g. see here: https://github.com/envoyproxy/gateway/blob/main/site/content/en/latest/user/circuit-breaker.md 

So, you can probably avoid the `httpbin` installation prerequisite and base this demo on the quickstart example, making this a bit simpler for users.

Reply:
👌